data/markers.json

```diff
@@ -28621,5 +28621,68 @@
 		"x": 33954,
 		"y": 32076,
 		"z": 15
+	},
+	{
+		"description": "Water buffalo (spawn location)",
+		"icon": "star",
+		"x": 32820,
+		"y": 32168,
+		"z": 7
+	},
+	{
+		"description": "Water buffalo (spawn location)",
+		"icon": "star",
+		"x": 32863,
+		"y": 32141,
+		"z": 7
+	},
+	{
+		"description": "Water buffalo (spawn location)",
+		"icon": "star",
+		"x": 32876,
+		"y": 32199,
+		"z": 7
+	},
+	{
+		"description": "Water buffalo (spawn location)",
+		"icon": "star",
+		"x": 32893,
+		"y": 32236,
+		"z": 7
+	},
+	{
+		"description": "Water buffalo (spawn location)",
+		"icon": "star",
+		"x": 32948,
+		"y": 32260,
+		"z": 7
+	},
+	{
+		"description": "Water buffalo (spawn location)",
+		"icon": "star",
+		"x": 32962,
+		"y": 32223,
+		"z": 7
+	},
+	{
+		"description": "Water buffalo (spawn location)",
+		"icon": "star",
+		"x": 33060,
+		"y": 32143,
+		"z": 7
+	},
+	{
+		"description": "Water buffalo (spawn location)",
+		"icon": "star",
+		"x": 33093,
+		"y": 32076,
+		"z": 7
+	},
+	{
+		"description": "Dwarf Mines",
```

Comment:
This marker was already present at the exact same position, with the label "Dwarfs". :/

Reply:
Whoops! I must have used a map version from before it was added, then added it myselfl